source/github/events/github_runner.ts

```diff
@@ -177,6 +177,7 @@ export const runEverything = async (
   const eventRuns = runs.filter(r => r.dslType === RunType.import)
 
   // Loop through all PRs, which are require difference DSL logic compared to simple GH webhook events
+  // TODO a re-mapped PR from an issue event won't have pr.head so how do we handle that in this method?
```

Comment:
@orta so the issue with my approach here is that the payload from github on the event does not contain the ".head" node.   This would be the same issue I'll run into for issue #342 as well.   Is there a means to do a "blind" trigger of a PR in peril?